app/controllers/planning_application/confirm_press_notices_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class PlanningApplication
+  class ConfirmPressNoticesController < AuthenticationController
+    before_action :set_planning_application
+    before_action :set_press_notice, only: %i[edit update]
+    before_action :ensure_press_notice_is_editable, only: %i[edit update]
+
+    def edit
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def update
+      respond_to do |format|
+        if @press_notice.update(press_notice_params)
```

Comment:
doesn't this normally go on the outside of the respond_to block?

Reply:
It's inconsistent as I think we do both styles in the app but this way enables us to just have the one block and also have the else to use the format
```
else
    format.html { render :edit }
 end
```